### PR TITLE
[CCI] CumulusCI 3.19.0 release

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Review by codexmarcel — CHANGES_REQUESTED** [review]
Wonky formatting on `HISTORY.rst`

File: HISTORY.rst [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+  - The task create_permission_set allows for creating and assigning a Permission Set that enables specific User Permissions. (Note: other types of permissions are not yet supported).
+  - The task create_bulk_data_permission_set creates a Permission Set with the Hard Delete and Set Audit Fields permissions for use with data load operations. The org permission to allow Set Audit Fields must be turned on.
+  - CumulusCI's load_dataset and extract_dataset tasks now support relative dates. To take advantage of relative dates, include the anchor_date key (with a date in YYYY-MM-DD format) in each mapping step you wish to relativize. On extract, dates will be modified to be the same interval from the anchor date as they are from the current date; on load, dates will be modified to be the same interval from today's date as they are from their anchor. Both date and date-time fields are supported.
+
+- Other:
+
+  - Small refactor to use fstring and breakdown the large _get_branch_tree() method into more manageable pieces.
```
Comment (davisagli) [inline HISTORY.rst:18]: This is an internal refactor that shouldn't be called out in the release notes.

File: HISTORY.rst [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+  - The oid_as_pk key is no longer supported in bulk data mappings. (This key was already deprecated). Select object Id mode by including the Id field in mappings.
+
+- Issues closed:
+
+  - We fixed an issue (#2001) that caused CumulusCI to extract invalid data sets when using after: steps with autoincrement primary keys.
+  - Fixed an issue where the retrieve_changes task did not actually retrieve folders.
+  - Fix a bug in the metadeploy_publish task where labels starting with "Install " were not extracted for localization.
+  - Fixed a bug that prevented using JWT auth with sandboxes if the sandbox's instance_url did not include an instance name.
+  - Fixed Account Table referencing lookup spelling error.
```
Comment (davisagli) [inline HISTORY.rst:27]:
```suggestion
  - Fixed a bug where ``cci project init`` generated an invalid mapping for bulk data tasks.
```

File: HISTORY.rst [hunk 3]
```diff
@@ -1,6 +1,29 @@
 =======
 History
 =======
+3.19.0 (2020-09-17)
```
Comment (davisagli) [inline HISTORY.rst:4]: I see we've missed this the past few releases, but the release number should have a line under it to turn it into a heading on https://cumulusci.readthedocs.io/en/latest/history.html

Like this: https://github.com/SFDO-Tooling/CumulusCI/pull/2030/files#diff-d9ae695c8119e58c2ec435b6deae2d53R107-R108

**Review by codexmarcel — DISMISSED** [review]

**Review by codexmarcel — APPROVED** [review]
diff looks good

**Review by davisagli — APPROVED** [review]

**Pull request merged**